RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 - [*] Fix: Product variations only support at most one image, so we won't show an option to add a second one. [https://github.com/woocommerce/woocommerce-ios/pull/3994]
 - [*] Fix: The screen to select images from the Media Library would sometimes crash when the library had a specific number of images. [https://github.com/woocommerce/woocommerce-ios/pull/4003]
+- [*] Improved error messages for logins.
```

Comment:
nit: add link to this PR?

Reply:
Lol can you tell that I don't know how to contribute code anymore? 🤣 Thanks for catching it! Fixed in 8ff08d8e0e7c888e205447a310a18b8110b8ae24.